legal-api/src/legal_api/resources/v2/business/business_documents.py

```diff
@@ -95,34 +96,35 @@ def _get_document_list(legal_entity):
 
 
 # This is used as part of entity snapshot in business-edit-ui.
-def _get_coop_documents_and_info(legal_entity):
+def _get_coop_documents_and_info(business):
     """Get certified memorandum and rules documents + info for coop."""
     documents, info = {}, {}
 
-    if not legal_entity:
+    if not business:
         return documents, info
 
     base_url = current_app.config.get("LEGAL_API_BASE_URL")
     base_url = base_url[: base_url.find("/api")]
 
     info["certifiedRules"], info["certifiedMemorandum"] = {}, {}
 
-    sr_filings = Filing.get_filings_by_types(legal_entity.id, ["specialResolution"])
+    business_id = business.id if business.is_legal_entity else business.legal_entity_id
```

Comment:
The business passed into `_get_coop_documents_and_info` will always be a `LegalEntity` instance as this function is specific to a COOP.  i.e. you can just use business.id

Reply:
updated